canonical-web-and-design/maas-ui
# feat(ui): build overall LXD resources card

## Done

- Created a new component `LxdResources` which houses all the content under the "Resources" tab for LXD KVMs. I decided not to use the existing `KVMResources` component because there are just too many differences.
- Built LXD overall resources card according to the [design](https://app.zeplin.io/project/601454d5d1141e1b6ea271b7/screen/602aaa0cf4fce564ee6b02c5)
- Added a placeholder component `ProjectResourcesCard` for the project's resources and renamed the original `ProjectResourcesCard` to `ProjectSummaryCard` (hoping it's less confusing to have all the `...Resources` components live under the "Resources" tab)

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the "Resources" tab for a LXD KVM and check that the Overall section matches the [design](https://app.zeplin.io/project/601454d5d1141e1b6ea271b7/screen/602aaa0cf4fce564ee6b02c5).

## Fixes

Fixes #2227 

## Screenshot
![Screenshot_2021-04-06 LXD resources bolla bolla MAAS(1)](https://user-images.githubusercontent.com/25733845/113664779-b6040900-96ef-11eb-9de4-238a6be1cc04.png)
